code/game/objects/structures/barsign.dm

```diff
@@ -54,7 +54,7 @@
 	icon_state = "Off"
 	anchored = 1
 	var/cult = 0
-	req_access = list(access_bar) //Has to initalize at first, this is updated by instance's req_access
+	req_one_access = list(access_bar, access_kitchen) //Has to initalize at first, this is updated by instance's req_access
```

Comment:
This does nothing, as the code comment says the req_access variable in the object code does nothing, and is overridden by the other access. This possibly would break the space bar barsign.

Instead of this, edit the req_access in the kitchen barsign specificially, and it would work.

Reply:
Yes, it does something and it would've worked but so you dont get buttblasted any more, it has now been done properly in a Not-2AM-fashion